### PR TITLE
Add DBeaver

### PR DESCRIPTION
**Pull request by lucasdaiki**
é nois memo

### CODE REVIEW

**Comment by ErickWendel** [comment]
Hey bro, could you add the "echo installing dbeaver" to the sccript?

File: startup.sh [hunk 1]
```diff
@@ -203,4 +203,9 @@ curl -s https://brave-browser-apt-release.s3.brave.com/brave-core.asc | sudo apt
 source /etc/os-release
 echo "deb [arch=amd64] https://brave-browser-apt-release.s3.brave.com/ $UBUNTU_CODENAME main" | sudo tee /etc/apt/sources.list.d/brave-browser-release-${UBUNTU_CODENAME}.list
 sudo apt update
-sudo apt install brave-keyring brave-browser
\ No newline at end of file
+sudo apt install brave-keyring brave-browser
+
+echo 'installing dbeaver'
```
Comment (lucasdaiki) [inline startup.sh:208]: @ErickWendel it's already here :)

**Review by lucasdaiki — COMMENTED** [review]
https://github.com/ErickWendel/ew-ubuntu-setup/pull/7/files#diff-d0486b0d7b181591f52bad014cd3a06fR208

**Review by ErickWendel — APPROVED** [review]

**Comment by ErickWendel** [comment]
:shipit:

**Pull request merged**